Sources/Testing/Events/Recorder/Event.JUnitXMLRecorder.swift

```diff
@@ -183,14 +209,19 @@ extension Event.JUnitXMLRecorder {
   ///
   /// - Returns: `character`, or a string containing its escaped form.
   private static func _escapeForXML(_ character: Character) -> String {
-    if character == #"""# {
+    switch character {
```

Comment:
`<` and `>` aren't _required_ to be escaped in XML attributes, but `&` definitely is. Could you add that?

Reply:
Gonna leave `>` out of pedantry, but yeah I'll add `&`.